personio-personnel-data-api-oa3.yaml

```diff
@@ -2191,27 +2197,33 @@ components:
     AttendanceCreateRequest:
       type: object
       properties:
-        employee:
-          example: 1
-        date:
-          type: string
-          format: date
-        start_time:
-          type: string
-          pattern: ^\d\d:\d\d$
-          example: "08:00"
-          description: "Format: hh:mm"
-        end_time:
-          type: string
-          pattern: ^\d\d:\d\d$
-          example: "17:00"
-          description: "Format: hh:mm"
-        break:
-          type: integer
-          example: 60
-        comment:
-          type: string
-          example: "I was productive as hell"
+        attendances:
+          type: array
+          properties:
```

Comment:
I believe you need to have an `items:` and then the properties nested under it. I am using this: https://editor.swagger.io/ to validate it btw

Reply:
You are right, I have to indicate it's an array of objects, changing it.